### ISSUE TITLE
import: improve error reporting for duplicate keys

### ISSUE THREAD

**Issue by couchand** [cockroachdb/cockroach#25387]
When an index has multiple keys, it would be helpful to share specifically which record caused the problem, to help the user debug the issue.

From **@thstart** on [Gitter](https://gitter.im/cockroachdb/cockroach?at=5af341d42bdad8046d5e273f):

> I just did IMPORT / caddy and it Failed. what to look for why it happened? The message from command line was: Error: pq: SST creation error at /Table/54/1/"010010201001000_257291498"/0; this can happen when a primary or unique index has duplicate keys: Invalid argument: Keys must be added in order. If in this message there is an info which record number coused the problem I can now better.

Jira issue: CRDB-5716

**Comment by nstewart** [comment]
I can reproduce this by `dump`ing tpcc with `--dump-mode schema` and then doing a CSV EXPORT of the "order" table (`WITH chunk_rows = '5000' FROM SELECT * FROM "order"`). 

When I point IMPORT at the schema and files 0,1, and 2 (note, I didn't include the later sequential files; was surprised this would matter), I get `pq: SST creation error at /Table/64/1/0/2/-1046/0; this can happen when a primary or unique index has duplicate keys: Invalid argument: Keys must be added in order`

**Comment by nstewart** [comment]
@mjibson FYI: When going through the repro steps again, I couldn't reproduce the original error on IMPORT. I did see that if you have many csv files it's pretty easy to accidentally use the same n1.* filename twice, which would also generate this error. That could have been what happened when I first tried (though I'm pretty sure I checked that).

This could have been user error but either way, +1 on improving the error message.
